soc/arm/ti_simplelink/cc13x2x7_cc26x2x7/soc.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Vaishnav Achath
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef TI_SIMPLELINK_CC13X2_CC26X2_SOC_H_
+#define TI_SIMPLELINK_CC13X2_CC26X2_SOC_H_
+
+/* CMSIS required values */
```

Comment:
Zephyr has a CMSIS module - could this be pulled in from elsewhere, avoiding the additional typedef?

That's the last compliance issue.